defi/src/storeGetCharts.ts

```diff
@@ -263,10 +295,21 @@ const handler = async (_event: any) => {
     })
   );
 
-  const dataFalseTrue = getHistoricalTvlForAllProtocols(false, true);
-  const dataFalseFalse = getHistoricalTvlForAllProtocols(false, false);
+  const dataFalseTrue = getHistoricalTvlForAllProtocols(false, true, options);
+  const dataFalseFalse = getHistoricalTvlForAllProtocols(false, false, options);
   await storeR2JSONString("cache/getHistoricalTvlForAllProtocols/false-true.json", JSON.stringify(await dataFalseTrue))
   await storeR2JSONString("cache/getHistoricalTvlForAllProtocols/false-false.json", JSON.stringify(await dataFalseFalse))
+
+  if (options.usePGCache) {
```

Comment:
why store this only when usePGCache?

Reply:
If we are not using PG cache, this can be expensive as each time we pull all daily data, last hourly data and import module for each protocol

But I have added a cache in this commit now, so removing this check in this commit: https://github.com/DefiLlama/defillama-server/pull/4976/commits/4755594552f0a7a135e41d9b30f13223dfaad981